src/NUnitFramework/framework/Internal/Randomizer.cs

```diff
@@ -61,7 +63,7 @@ static Randomizer()
 
         // Static Random instance used exclusively for the generation
         // of seed values for new Randomizers.
-        private static Random _seedGenerator;
+        private static Random _seedGenerator = null!; // Initialized by the InitialSeed setter called by the static constructor.
```

Comment:
`null!` is new to me. Can you explain? I tried to Google it but I couldn't find it. Thx.

Reply:
Ah, the `!` part means "suppress the nullability warning." I think of the null forgiveness operator `!` as being an inline pragma.

`= null!` tells the compiler, "trust me, this field is always initialized before being accessed." It's necessary in this case because the static constructor doesn't directly set the field, and the compiler has not been given the smarts to analyze across more than one method body at a time. But I believe this is being worked on, so one day we will probably be able to remove the `= null!`.

Another approach would be to refactor so that `_seedGenerator` is directly initialized in the static constructor.

Reply:
Thanks. You have your one review if you want to merge. 👍

Reply:
Cool, I do want to get this in because TestContext is a user-facing piece.